data/tools/Android.json

```diff
@@ -0,0 +1,40 @@
+{
+  "tools": [
+  
+      {
+      "name": "Andriller",
+      "source": "https://github.com/den4uk/andriller",
+      "description": "Software utility with a collection of forensic tools for smartphones.",
```

Comment:
There is no Android focused category, but android reverse tools can be put in the reverse category, androids forensics tools in the forensics category and so on
So just put this one in Digital Forensics please.